compiler/utils/string_common.ml

```diff
@@ -0,0 +1,51 @@
+(* This file is part of the Catala compiler, a specification language for tax
+   and social benefits computation rules. Copyright (C) 2020 Inria, contributor:
+   Denis Merigoux <denis.merigoux@inria.fr>, Emile Rolley <emile.rolley@tuta.io>
+
+   Licensed under the Apache License, Version 2.0 (the "License"); you may not
+   use this file except in compliance with the License. You may obtain a copy of
+   the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+   License for the specific language governing permissions and limitations under
+   the License. *)
+
+let to_ascii : string -> string = Ubase.from_utf8
+
+let is_uppercase (c : char) : bool =
+  let c = Char.code c in
+  0x41 <= c && c <= 0x5b
```

Comment:
This should be renamed `is_uppercase_ascii`, because it doesn't work with UTF8 chars.